package.json

```diff
@@ -6,19 +6,21 @@
   "homepage": "https://github.com/CityOfZion/neon-wallet",
   "author": "Ethan Fast <ejhfast@gmail.com> (https://github.com/Ejhfast)",
   "scripts": {
-    "postinstall": "electron-builder install-app-deps",
+    "postinstall": "electron-builder install-app-deps && npm run electron-rebuild",
```

Comment:
electron-builder install-app-deps && yarn run electron-rebuild

Reply:
I get a blank screen when i change this. It's related to the ABI mismatch.

only command that seems to make it run correctly is `npm install;webpack;npm run start-dev`.

this is because yarn does not build node-hid correctly.

i was unable to find a project that uses yarn and node-hid, only npm :(